src/Profile/PageProfile.cpp

```diff
@@ -40,47 +40,47 @@ static void
 Load(const ProfileMap &map, PageLayout &_pl, const unsigned page)
 {
   char profileKey[32];
-  unsigned prefixLen = sprintf(profileKey, "Page%u", page);
-  if (prefixLen <= 0)
-    return;
-
-  PageLayout pl = PageLayout::Default();
-  strcpy(profileKey + prefixLen, "InfoBoxMode");
-  if (!map.Get(profileKey, pl.infobox_config.auto_switch))
-    return;
-  strcpy(profileKey + prefixLen, "InfoBoxPanel");
-  if (!map.Get(profileKey, pl.infobox_config.panel))
-    return;
-
-  strcpy(profileKey + prefixLen, "Layout");
-  unsigned temp = 0;
-  map.Get(profileKey, temp);
-  switch (temp) {
-  case tlEmpty:
-    pl.valid = false;
-    break;
-
-  case tlMap:
-    pl.infobox_config.enabled = false;
-    break;
+  int prefixLen = snprintf(profileKey, 32, "Page%u", page);
```

Comment:
Why `snprintf()` instead of `sprintf()`? What's the point?
And why the hard-coded magic value?